src/strace.rs

```diff
@@ -0,0 +1,22 @@
+cfg_if::cfg_if! {
```

Comment:
FWIW this `cfg_if` call probably isn't needed if `init_strace` starts with `if !cfg!(...) { return }`